engines/io_uring.c

```diff
@@ -185,6 +186,15 @@ static struct fio_option options[] = {
 		.category = FIO_OPT_C_ENGINE,
 		.group	= FIO_OPT_G_IOURING,
 	},
+	{
+		.name	= "nowait",
+		.lname	= "No Wait",
```

Comment:
Maybe use the same lname as below: RWF_NOWAIT (or the later ones the same as this)?